plugins/GSdx/GSdx.rc

```diff
@@ -175,7 +181,9 @@ STYLE DS_SETFONT | DS_MODALFRAME | DS_FIXEDSYS | WS_POPUP | WS_CAPTION | WS_SYSM
 CAPTION "Settings..."
 FONT 8, "MS Shell Dlg", 400, 0, 0x1
 BEGIN
-    CONTROL         IDB_LOGO9,IDC_LOGO9,"Static",SS_BITMAP,7,7,175,44
+    CONTROL         IDB_PSX_LOGO9,IDC_PSX_LOGO9,"Static",SS_BITMAP | SS_REALSIZECONTROL,7,7,175,42
```

Comment:
It was done to have the height constant for all the logos. It was weird to have only a single logo with a different height.